packages/graphic-walker/src/fields/filterField/tabs.tsx

```diff
@@ -292,19 +326,72 @@ export const FilterTemporalRangeRule: React.FC<RuleFormProps & { active: boolean
         });
     }, []);
 
+    const dateStringFormatter = (timestamp: number) => {
+        return format(new Date(timestamp), "yyyy-MM-dd'T'HH:mm:ss")
+    }
+
     return field.rule?.type === 'temporal range' ? (
-        <Container>
-            <Slider
-                min={min}
-                max={max}
-                value={field.rule.value}
-                onChange={handleChange}
-                isDateTime
-            />
+        <Container className="overflow-visible">
+            <div>{t_filter_type('temporal_range')}</div>
+            <div className="text-gray-500">{t_filter_type('temporal_range_desc')}</div>
+            <CalendarInputContainer>
+                <div className="calendar-input">
+                    <div className="my-1">{t('range.start_value')}</div>
+                    <input
+                        className="block w-full rounded-md border-0 py-1 px-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-1 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
+                        type="datetime-local" 
+                        id="birthdaytime" 
+                        name="birthdaytime"
+                        min={dateStringFormatter(min)}
+                        max={dateStringFormatter(max)}
+                        defaultValue={field.rule.value[0] ? dateStringFormatter(field.rule.value[0]) : dateStringFormatter(min)}
+                        onChange={(e) => {
```

Comment:
mark

Reply:
i've encapsulated calendar input to a separate component